scripts/build-doc

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+/* eslint-env node */
+
+require('./util/work-from-root')();
+
+var docBuilder = require('./util/doc-builder')({
```

Comment:
This is just for the project itself though, right? I'd probably set a version of node you expect to be run locally and use that to give you access to some of the more modern features.